src/mixins/filing-template-mixin.ts

```diff
@@ -599,55 +600,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setResolutionDates([])
     this.setOriginalResolutionDates(businessSnapshot.resolutions)
 
-    // Store business contact
-    this.setBusinessContact(businessSnapshot.contactPoint)
-  }
-
-  /**
-    * Ensure consistent object structure for an incorporation application
-    * whether it contains a Name Request or not, and whether it is an initial
-    * draft or it has been previously saved. Object merging does not
-    * work very well otherwise (due to nested properties).
-    * @param filing the draft filing fetched from legal-api
-    * @returns the filing in safe-empty state if applicable
-  */
-  formatEmptyFiling (filing: any): IncorporationFilingIF {
```

Comment:
This wasn't used so I deleted it.

Reply:
Do you recall what we used this for initially? Was this a carry over from Create UI?

Reply:
I believe so.

And I think YOU implemented this :) as a way to pre-populate a filing.

Reply:
hahah fair enough. 
I have the memory of a peanut with everything going on these days!